ocfweb/main/templates/staff-hours.html

```diff
@@ -9,11 +9,11 @@
         <p>OCF staff members hold regular drop-in staff hours to provide assistance with account issues or with OCF services. We're always happy to help troubleshoot account or service issues!</p>
         <p>Keep in mind the OCF volunteers sometimes have last-minute conflicts, so it's a good idea to check this page before coming in for cancellations.</p>
         {% for staff_hour in staff_hours %}
-            <div class="hour {% if staff_hour.cancelled %} cancelled {% endif %}">
+            <div class="hour {% if staff_hour.cancelled or staff_hour.day == today and lab_status.force_lab_closed %} cancelled {% endif %}">
                 <div class="title">
                     <strong>{{staff_hour.day}}</strong>
                     {{staff_hour.time}}
-                    {% if staff_hour.cancelled %}
+                    {% if staff_hour.cancelled or staff_hour.day == today and lab_status.force_lab_closed %}
```

Comment:
nit: parens around `or staff_hour.day == today and lab_status.force_lab_closed`? order of operations is hard